cmake/imex.cmake

```diff
@@ -4,11 +4,10 @@ get_property(IMEX_INCLUDES GLOBAL PROPERTY IMEX_INCLUDES)
 if (NOT DEFINED IMEX_INCLUDES)
     include(functions)
     set(IMEX_CHECK_LLVM_VERSION ON)
-    set(IMEX_ENABLE_L0_RUNTIME 0)
     # TODO: Change to main https://github.com/oneapi-src/oneDNN.git when all the
     # required functionality is merged.
     gc_fetch_content(imex 496b240093b5e132b60c5ee69878300fe69be300 https://github.com/Menooker/mlir-extensions
-            CMAKE_ARGS "-DMLIR_DIR=${MLIR_DIR};-DIMEX_CHECK_LLVM_VERSION=ON;-DIMEX_ENABLE_L0_RUNTIME=0"
+            CMAKE_ARGS "-DMLIR_DIR=${MLIR_DIR};-DIMEX_CHECK_LLVM_VERSION=ON;-DIMEX_ENABLE_L0_RUNTIME=${IMEX_ENABLE_L0_RUNTIME}"
```

Comment:
we need l0 runtime for e2e tests, so we want to enable it sometimes

Reply:
Is it possible to use opencl? I think it should work. see `gpu-runner/mlp.mlir`

Reply:
Yes it's possible to use opencl. Replaced L0 with opencl